src/ioc.ts

```diff
@@ -1,14 +1,6 @@
 import { IocContainer } from '@tsoa/runtime'
 import { container } from 'tsyringe'
 
-import { logger, Logger, type ILogger } from './logger'
```

Comment:
moved this around as it was causing issues in tests because of order of operations